src/www/ui/api/Controllers/JobController.php

```diff
@@ -40,6 +41,20 @@
  */
 class JobController extends RestController
 {
+  public function getAllJobs($request, $response, $args)
+  {
+    $limit = 0;
+    $page = 1;
+    $page = $request->getHeaderLine('page');
```

Comment:
The `$limit` is still hardcoded. And the `$page` is assigned twice ???

Reply:
Sorry My bad. I fixed it.